plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -79,6 +80,7 @@ func tokenErrorf(s *api.Secret, format string, i ...interface{}) {
 //       token-id: ( token id )
 //       # Required key usage.
 //       usage-bootstrap-authentication: true
+//       auth-groups: "group1,group2"
```

Comment:
`system:bootstrappers:custom-group1.system:bootstrappers:custom-group1` for clarity